docs/installation/updating-with-npm.md

```diff
@@ -0,0 +1,31 @@
+# Updating GOV.UK Frontend with node package manager (NPM)
+
+Follow these instructions if you have previously installed the GOV.UK Frontend npm package following the [installation instructions](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#requirements), and want to update to a more recent version.
+
+## Steps to update
+
+### 1. Find out changes introduced in recent versions.
+
+To do this, view `package.json` at the root of your project to find out the version of govuk-frontend that your app is currently running, for instance "2.11.0". See [example](https://github.com/alphagov/govuk-design-system/blob/master/package.json#L34) for where this information lives in GOV.UK Design System.
```

Comment:
If we want to include an example could consider inlining a snippet rather than pointing to a file that can change.

However, If we keep pointed to the design system code base it should be permalinked.

Reply:
What about asking people to run `npm list govuk-frontend` instead?

```
$ npm list govuk-frontend
govuk_design_system@ /Users/oliverbyford/Code/govuk-design-system
└── govuk-frontend@2.11.0
```

Reply:
Thanks @36degrees. We decided to keep both methods in case someone doesn't have access to the terminal.